packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -30,14 +31,14 @@ interface ModalWrapperProps extends ModalProps {
   isOpen?: boolean
 }
 
-export function Modal(props: ModalProps) {
+function Modal(props: ModalProps, ref: DOMRef<HTMLDivElement>) {
```

Comment:
Do ModalProps need to be moved to the type package? My two modal questions also apply to tray?